src/routers/alpha-router/alpha-router.ts

```diff
@@ -2011,7 +2019,7 @@ export class AlphaRouter
     quoteToken: Token,
     providerConfig?: GasModelProviderConfig
   ): Promise<
-    [IGasModel<V3RouteWithValidQuote>, IGasModel<MixedRouteWithValidQuote>]
+    [IGasModel<V2RouteWithValidQuote> | undefined, IGasModel<V3RouteWithValidQuote>, IGasModel<MixedRouteWithValidQuote>]
```

Comment:
Especially now that it's possible that a V2 model isn't available, I'd rather us change this return type to 
```
{
v2GasModel?: IGasModel<V2RouteWithValidQuote>,
v3GasModel?: ...
}
```
Which provides cleaner restructuring IMO than arrays especially when dealing with optional values